webapp/src/main/resources/properties/en.properties

```diff
@@ -222,6 +222,12 @@ search.result.empty=There are no results. Update your search parameter to find w
 # Message displayed when the user has not selected repository and locales
 search.result.selectRepoAndLocale=Select at least one repository and locale before you can search.
 
+search.viewMode.title=View mode
+
+search.viewMode.FULL=Full
+search.viewMode.REDUCED=Reduced
```

Comment:
Reduced -> Standard?

Reply:
Maybe I just add a 4th standard state (and make it the default) which has the same information as the current layout?

Reply:
This is now implemented